doc/source/api/api.rst

```diff
@@ -8,6 +8,7 @@ The following API documentation describes how to use Covalent.
 - :ref:`electrons_api` and :ref:`lattices_api` are used for constructing workflows
 - :ref:`results_api` is used for collecting and manipulating results
 - :ref:`dispatcher_api` is used for dispatching workflows
+- :ref:`local_executor` is used to execute electrons locally.
```

Comment:
Either we need there to be `.` after all the lines or none of the lines for the sake of consistency. My bad, I think it was my suggestion to add the `.` int he first place.